### PR TITLE
feat(json-beautifier): allow single quote around values

### LINKED ISSUE #23: Support single or double quote around the properties
#### Instructions for filing an issue.

In the *JSON beautifier* app it would be nice if we can set whether to use single or double quote around the properties.

Let me know if this is something you feel you will like to know, I do not mind sending a PR.

### PR DESCRIPTION
**Pull request by hisabimbola**
This PR allows the user to select whether to allow single quote or double quote around values

fixes #23

### CODE REVIEW

File: js/csvjson/json_beautifier.js [hunk 1]
```diff
@@ -75,11 +75,12 @@
 			dropQuotesOnNumbers = options.dropQuotesOnNumbers || false,
 			inlineShortArrays = options.inlineShortArrays || false,
 			inlineShortArraysDepth = options.inlineShortArraysDepth || 1;
```
Comment (martindrapeau) [inline js/csvjson/json_beautifier.js]:
```
inlineShortArraysDepth = options.inlineShortArraysDepth || 1,
quoteType = options.quoteType || 'double';
```

File: application/views/json_beautifier_view.php [hunk 2]
```diff
@@ -54,6 +54,13 @@
 							<option value="..">..</option>
 						</select>
 					</label>
+					<label class="inline" title="Quote type around values">
+						Quotes
+						<select id="quote-type" name="quote-type" class="save">
+							<option value="single">Single Quote</option>
```
Comment (martindrapeau) [inline application/views/json_beautifier_view.php]: Simply use `Single` and `Double` for the option text - shorter.

**Comment by martindrapeau** [comment]
I tested it works nicely. Two small tweaks to make then I will merge your PR. Good job!

**Comment by hisabimbola** [comment]
Thanks for your review @martindrapeau 

Update the code.

**Pull request merged**